<launch>
  <arg name="marker_size" default="4.4" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="output_frame" default="/base" />

  <arg name="cam_image_topic_right" default="/cameras/right_hand_camera/image" />
  <arg name="cam_info_topic_right" default="/cameras/right_hand_camera/camera_info" />
  <node name="ar_track_alvar_right" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic_right) $(arg cam_info_topic_right) $(arg output_frame)">
    <remap from="ar_pose_marker" to="ar_pose_marker/right" />
  </node>
  
  <arg name="cam_image_topic_left" default="/cameras/left_hand_camera/image" />
  <arg name="cam_info_topic_left" default="/cameras/left_hand_camera/camera_info" />
  <node name="ar_track_alvar_left" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic_left) $(arg cam_info_topic_left) $(arg output_frame)">
    <remap from="ar_pose_marker" to="ar_pose_marker/left" />
  </node>
</launch>

